switchapi/es2k/switch_fdb.c

```diff
@@ -298,7 +298,8 @@ switch_status_t switch_api_l2_forward_create(
         api_tunnel_info = &tunnel_info->api_tunnel_info;
     }
 
-    if (api_l2_info->learn_from == SWITCH_L2_FWD_LEARN_PHYSICAL_INTERFACE) {
+    if (api_l2_info->learn_from == SWITCH_L2_FWD_LEARN_PHYSICAL_INTERFACE
```

Comment:
one line comment above this, on what this condition does and why we added SWITCH_RIF_HANDLE

Reply:
Addressed.